pysat/tests/test_instruments.py

```diff
@@ -204,3 +200,32 @@ def test_load(self, inst, clean_level):
                 remove_files(inst)
         else:
             pytest.skip("Download data not available")
+
+
+class TestInstrumentsNoDownload():
+
+    def setup(self):
+        """Runs before every method to create a clean testing setup."""
+        self.package = 'pysat.instruments'
+
+    def teardown(self):
+        """Runs after every method to clean up previous testing."""
+        del self.package
+
+    @pytest.mark.parametrize("inst", instruments['no_download'])
+    def test_download_warning(self, inst):
+        print(' '.join(('\nChecking download routine warnings for module: ',
+                        inst.platform, inst.name, inst.tag, inst.sat_id)))
+        start = inst._test_dates[inst.sat_id][inst.tag]
+        with warnings.catch_warnings(record=True) as war:
+            try:
+                inst.download(start, start)
+            except ValueError as strerr:
+                if str(strerr).find('CDAAC') >= 0:
```

Comment:
What's this about?

Reply:
This is temporary code that will be resolved when I get the cosmic stuff resolved.  Cosmic falls into the in between zone: can't be tested with the download codes, doesn't have an appropriate error message.  A new set of flags should fix this.